src/bin/add/fetch.rs

```diff
@@ -323,30 +317,27 @@ pub fn get_crate_name_from_path(path: &str) -> Result<String, FetchGitError> {
 }
 
 fn get_name_from_manifest(manifest: &Manifest) -> Result<String, FetchGitError> {
-    manifest.data
+    manifest
+        .data
         .get("package")
         .and_then(|m| m.get("name"))
         .and_then(|name| name.as_str().map(|s| s.to_string()))
         .ok_or(FetchGitError::ParseCargoToml)
 }
 
-fn get_cargo_toml_from_git_url(url: &str) -> Result<String, FetchGitError> {
-    let mut easy = Easy::new();
-    easy.url(url)?;
-    easy.get(true)?;
-    easy.accept_encoding("text/plain")?;
-
-    let mut html = Vec::new();
-    {
-        let mut transfer = easy.transfer();
-        transfer.write_function(|data| {
-            html.extend_from_slice(data);
-                    Ok(data.len())
-        })?;
-
+fn get_default_timeout() -> Duration {
+    Duration::from_secs(10)
```

Comment:
10 seconds sounds good. What's the error message we get when the timeout was hit?

(Also, can you add new changes as separate commits in the future? We can always squash them later.)